pom.xml

```diff
@@ -169,7 +169,7 @@
         <jakarta.mvc.version>2.1.0</jakarta.mvc.version>
         <krazo.version>3.0.1</krazo.version>
         <yasson.version>3.0.4</yasson.version>
-        <eclipselink.version>5.0.0-B02</eclipselink.version>
+        <eclipselink.version>5.0.0-B07</eclipselink.version>
         <eclipselink.asm.version>9.7.1</eclipselink.asm.version>
```

Comment:
Did you miss this one?

Reply:
@svendiedrichsen in Payara7 we do not use a patch. So we need to wait the next release for version 5 which will include the asm version change.